cd/experimental/graph3.ocd

```diff
@@ -0,0 +1,147 @@
+<CD xmlns="http://www.openmath.org/OpenMathCD">
+  <CDName>graph3</CDName>
+  <CDBase>http://www.openmath.org/cd</CDBase>
+  <CDURL>http://www.openmath.org/cd/graph3.ocd</CDURL>
+  <CDReviewDate>2018-12-28</CDReviewDate>
+  <CDStatus>experimental</CDStatus>
+  <CDDate>2018-12-28</CDDate>
+  <CDVersion>0</CDVersion>
+  <CDRevision>1</CDRevision>
+  
+  
```

Comment:
This CD should have author and licensing information. Maybe use the official CDs as a template: eng. https://github.com/OpenMath/CDs/blob/master/cd/Official/alg1.ocd

Reply:
Thanks for your comment! I based my contribution on the other graph contributions, which also not include this info. Should this be OpenMath as author and copyright holder (as the official CDs), or myself?